<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QDialog" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>469</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>469</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки WordGame</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Подсказки</string>
   </property>
   <widget class="QCheckBox" name="showFirstLetter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Показывать первую букву слова</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Статистика</string>
   </property>
   <widget class="QCheckBox" name="statsTime">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Время</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statsEntered">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>45</y>
      <width>145</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во введенных слов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statsWins">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>69</y>
      <width>64</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Победы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetStats">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showStats">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>О программе</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>WordGame Русский, версия 1.0
Open Source проект
Сделано в России (2018 год)
Использует Qt 5.2.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Игра</string>
   </property>
   <widget class="QComboBox" name="gameMode">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>21</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Слова и геогр. назв.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Георафич. названия</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим игры</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Основное</string>
   </property>
   <widget class="QPushButton" name="exportSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Экспорт настроек</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importSettings">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Импорт настроек</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingsForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
